workspace/coordinates.ts

```diff
@@ -0,0 +1,32 @@
+const request = require('request')
+
+import { BusStop } from './busStop'
+
+export class Coordinates {
+    latitude: string;
+    longitude: string;
+
+    constructor(lat: string, lon: string){
+        this.latitude = lat; 
+        this.longitude = lon;
+    }
+    getStopPoints(): Promise <BusStop[]> {
+        let promiseLatAndLong: Promise <string> = new Promise((resolve) => {
+            var requestString = "https://api.tfl.gov.uk/StopPoint?stopTypes=NaptanPublicBusCoachTram&lat="+
+                this.latitude + "&lon=" + this.longitude //+ "?app_id=ef0cca24&app_key=281fde5472478ee61c2d8a63c5e4f554"
+            request(requestString, function (error, response, body) {
+                resolve(body); 
+            });
+        });
+
+        return promiseLatAndLong.then((bodyString: string) => {
+            var latLongArray = JSON.parse(bodyString);
+           //console.log(latLongArray.stopPoints);
```

Comment:
So, I know this is probably either (a) forgotten, or (b) used occasionally to check what's happening, but committing commented out code is Real Bad.

Basically, future readers have no idea if it's needed/briefly commented out/completely irrelevant, making it very difficult to move safely. I've worked in teams that do this, and it's real frustrating (I just deleted all of it I saw until they stopped). But yeah, worth watching out for.